### DISCUSSION TITLE
Why are the Training Stats much lower using Spacy compared to Prodigy

### DISCUSSION THREAD

**Discussion by DerDiego13** [explosion/spaCy#12594] [Help: Other Questions]
Hi together,

basically, the title describes my observation already. When I train a model in Prodigy, Scores are much higher much earlier on compared to training the same model with the same data in Spacy.

**Comment by adrianeboyd** [comment]
My first guess would be that it is related to config settings or data splits, but it's hard to know without more details about the exact configs and training commands, etc. from both prodigy and spacy?
